python/mlx/nn/layers/base.py

```diff
@@ -559,3 +559,24 @@ def eval(self):
         See :func:`train`.
         """
         self.train(False)
+
+    def set_dtype(
+        self,
+        dtype: mx.Dtype,
+        predicate: Optional[Callable[[mx.Dtype], bool]] = lambda x: mx.issubdtype(
+            x, mx.floating
+        ),
+    ):
+        """Set the dtype of the models parameters.
+
+        Args:
+            dtype (Dtype): the new dtype
+            predicate (~typing.Callable): a predicate to select parameters. By default, only parameters of type
+                :attr:`~mlx.core.floating` will be updated to avoid updating any integer parameters to the new dtype.
```

Comment:
```suggestion
            predicate (~typing.Callable): A predicate to select parameters to cast. By default, only parameters of type
                :attr:`~mlx.core.floating` will be updated to avoid updating integer parameters to the new dtype.
```